### PR TITLE
Fixed up sold out and feedback after event closed

### PR DESCRIPTION
**Pull request by robdmoore**
cc @robcrowley

### CODE REVIEW

File: pages/agenda.tsx [hunk 1]
```diff
@@ -46,15 +46,15 @@ class AgendaPage extends React.Component<AgendaPageProps> {
     const dates = this.props.pageMetadata.dates
 
     const AfterSessionDetails = (s: Session) => {
-      return (
+      return dates.AcceptingFeedback ? (
```
Comment (pzi) [inline pages/agenda.tsx]: Not a fan of?

```jsx
if (!dates.AcceptingFeedback) return null

return (
    <p className="text-center">
        <a className="btn btn-secondary" target="_blank" href={conference.SessionFeedbackLink}>
            Give feedback
        </a>
    </p>
)
```

Note, I also removed the unnecessary fragment as the `p` was a single-node return anyway.

Comment (robdmoore) [reply pages/agenda.tsx]: Sorry, missed this. Feel free to push that change onto my PR branch

Comment (robdmoore) [reply pages/agenda.tsx]: I've merged anyway, feel free to go back and correct :)

Comment (pzi) [reply pages/agenda.tsx]: Ha, forgot about this as well :) Will do!

**Pull request merged**